### PR TITLE
Ninja implants!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Ninja now gets two implants! - storage normal traitor implant to hold a few items inside themelfs

Combat arm implant that has a flash, dart gun and a crossbow for none lethal combat

## Why It's Good For The Game
Allows ninjas to be less lethal all the time and gives them more options to deal with threats!

## Changelog
:cl:
add: Ninja;s now get implants to help them complete their work in a less lethal way
/:cl:

### CODE REVIEW

**Review by Ghommie — CHANGES_REQUESTED** [review]
This goes against their design in the current. Their gameplay focus is around a special hi-tech outfit that grants them a range of abilities and a special sword they fight and teleport around with.
So if you want to give them something new, that probably should be something that complements better their nature like an additional ninja suit function, and not some recycled cybernetic gadget feature.

**Comment by silicons** [comment]
^
Return old ninja if you actually want to do this kinda stuff, Trilby.

**Comment by Trilbyspaceclone** [comment]
Idk how to give them non-lethal gear. 
I was told they were designed to be op like wizard but noticed they have no real way to do combat in a non-lethal way

**Comment by Arturlang** [comment]
They do have nonlethal weaponry, such as the set gun, after which they can simply stun hand the netted person, or simply crit them while not aiming for vital points, and then heal them back up once cuffed, using some medkit they got somewhere

File: code/modules/ninja/outfit.dm [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
 /datum/outfit/ninja/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE, client/preference_source)
+	var/obj/item/organ/I = /obj/item/organ/cyberimp/arm/ninja
+	I.Insert(src)
 	if(istype(H.wear_suit, suit))
 		var/obj/item/clothing/suit/space/space_ninja/S = H.wear_suit
 		if(istype(H.belt, belt))
 			S.energyKatana = H.belt
-		S.randomize_param()
-
+		S.randomize_param()
```
Comment (necromanceranne) [inline code/modules/ninja/outfit.dm:25]: Don't forget the newline here.

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 2]
```diff
@@ -266,6 +266,11 @@
 	volume = 10
 	proj_piercing = 1
 
+/obj/item/reagent_containers/syringe/piercing/ninja
```
Comment (necromanceranne) [inline code/modules/reagents/reagent_containers/syringes.dm:269]: I would recommend maybe giving ninjas a way to synthethize these so they aren't reliant on scavenging.

File: code/modules/ninja/ninja_event.dm [hunk 3]
```diff
@@ -90,4 +90,4 @@ Contents:
 	A.real_name = "[pick(GLOB.ninja_titles)] [pick(GLOB.ninja_names)]"
 	A.copy_to(new_ninja)
 	new_ninja.dna.update_dna_identity()
-	return new_ninja
+	return new_ninja
```
Comment (necromanceranne) [inline code/modules/ninja/ninja_event.dm:93]: This needs a newline.

Comment (Trilbyspaceclone) [reply code/modules/ninja/ninja_event.dm:93]: This is just do to editing the file once and then going back on it, nothing is changed

File: code/modules/ninja/outfit.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/ninja/outfit.dm:25]: Same as before, all files do this for me

File: code/modules/reagents/reagent_containers/syringes.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/reagents/reagent_containers/syringes.dm:269]: maybe fore 200E or something cuz its really powerfull

**Comment by Trilbyspaceclone** [comment]
Rng fail but none the less this pr is ahead of its time

**Pull request closed without merging**